### DISCUSSION TITLE
Cannot read properties of undefined   -- for all types in module

### DISCUSSION THREAD

**Discussion by npreiser** [redhat-developer/vscode-extension-tester#1505] [Q&A]
I have integrated the tester, and its compiling in just fine.  (version 8.6.0),  however, when I go to run a test, which utilizes any of the types ,  I get the same error,   "Cannot read properties of undefined".   I have confirmed that the module is there, and its actually resolving to the correct type  in the out dir..etc.  I can even breakpoint there .   This is happening for all type,  I am just trying to run some of the basic examples .    Has anyone seen this behavior before.   is this a webpack issue maybe?,

**Comment by npreiser** [comment]
for example ,  a call like :   VSBrowser.instance,    always returns undefined,   am I missing the correct chrome driver?,  or another dependency?

**Comment by djelinek** [comment]
could you please share details on yours setup? (OS, NodeJS, Java,) ? and maybe the way how are you trying to run tests?
